agent/tests/test_api_metadata.c

```diff
@@ -123,25 +126,25 @@ static void get_trace_metadata_tests_when_dt_enabled(zval* retval) {
 static void test_get_trace_metadata_when_dt_enabled(TSRMLS_D) {
   test_get_trace_metadata_helper(get_trace_metadata_tests_when_dt_enabled);
 }
+#endif
+
 void test_main(void* p NRUNUSED) {
 #if defined(ZTS) && !defined(PHP7)
   void*** tsrm_ls = NULL;
 #endif /* ZTS && !PHP7 */
 
   tlib_php_engine_create(
       "newrelic.distributed_tracing_enabled = false\n" PTSRMLS_CC);
-
   test_is_sampled(TSRMLS_C);
   test_get_linking_metadata_when_dt_disabled(TSRMLS_C);
   test_get_trace_metadata_when_dt_disabled(TSRMLS_C);
-
   tlib_php_engine_destroy(TSRMLS_C);
 
+#if 0
   tlib_php_engine_create("" PTSRMLS_CC);
```

Comment:
Turns out that creating second php engine within a test causes some issues when building with PHP 7.0, 7.1, 7.2, and 7.3 on Alpine Linux as well as on CentOS.